pkg/validate/validate.go

```diff
@@ -25,9 +29,37 @@ const kubeconformCommentFormat = `
 </details>
 `
 
-var schemaLocations = []string{
-	`./schemas/{{ .NormalizedKubernetesVersion }}/{{ .ResourceKind }}{{ .KindSuffix }}.json`,
-	"https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master/{{ .NormalizedKubernetesVersion }}-standalone{{ .StrictSuffix }}/{{ .ResourceKind }}{{ .KindSuffix }}.json",
+var localSchemasLocation string
+
+func getSchemaLocations() []string {
+	once.Do(func() {
+		schemasLocation := viper.GetString("schemas-location")
+		localSchemasLocation = "./schemas"
+		// if this is a git repo, we want to clone it locally
+		if strings.HasPrefix(schemasLocation, "https://") || strings.HasPrefix(schemasLocation, "git@") {
+
+			tmpSchemasLocalDir := "/tmp/schemas"
```

Comment:
rather than a hardcoded dir, can we use [os.MkdirTemp](https://pkg.go.dev/os#MkdirTemp)?